lib/wificom/wifi_nina.py

```diff
@@ -30,36 +28,28 @@ def __init__(self, esp32_cs, esp32_busy, esp32_reset, esp32_sck, esp32_mosi, esp
 		self.esp = adafruit_esp32spi.ESP_SPIcontrol(self.spi, self.esp32_cs, self.esp32_ready, \
 			 self.esp32_reset)
 		socket.set_interface(self.esp)
-
+	# pylint: disable=unused-argument,invalid-name,inconsistent-return-statements
 	def connect(self):
 		'''
 		Connect to a supported board's WiFi network
 		'''
-		print("Connecting to WiFi network [" + secrets_wifi_ssid + "]...")
-
-		secrets_payload = {
-			'ssid': secrets_wifi_ssid,
-			'password': secrets_wifi_password
-		}
-
-		wifi_manager = adafruit_esp32spi_wifimanager.ESPSPI_WiFiManager(self.esp, secrets_payload)
-
-		# Check status, attempt reconnect if connect attempt fails
-		try:
-			esp_status = self.esp.status
-			if esp_status == 0:
-				wifi_manager.connect()
-		except RuntimeError:
-			wifi_manager.reset()
-
-		print("Connected to WiFi network!")
-
-		mqtt_client = MQTT.MQTT(
-			broker=secrets_mqtt_broker,
-			username=secrets_mqtt_username.lower(),
-			password=secrets_mqtt_password,
-		)
-
-		MQTT.set_socket(socket, self.esp)
-
-		return mqtt_client
+		for network in secrets_wireless_networks:
+			#pylint: disable=unused-variable
```

Comment:
Use _?

Reply:
Fixed, put "network" back for wifi classes also